lang/python/python-wcwidth/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2019 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-wcwidth
+PKG_VERSION:=0.1.7
+PKG_RELEASE:=1
+
+PYPI_NAME:=wcwidth
+PKG_HASH:=3df37372226d6e63e1b1e1eda15c594bca98a22d33a23832a90998faa96bc65e
+PKG_BUILD_DIR:=$(BUILD_DIR)/wcwidth-$(PKG_VERSION)
```

Comment:
Setting `PKG_BUILD_DIR` shouldn't be necessary.

Reply:
Removed